udp_linux.go

```diff
@@ -87,8 +87,17 @@ func NewListener(ip string, port int, multi bool) (*udpConn, error) {
 	return &udpConn{sysFd: fd}, err
 }
 
-func (u *udpConn) Rebind() {
-	return
+func (u *udpConn) Rebind() error {
+	return nil
+}
+
+func (ua *udpAddr) Copy() udpAddr {
+	nu := udpAddr{
+		IP:   ua.IP,
+		Port: ua.Port,
+	}
+
+	return nu
```

Comment:
You should be able to just do `return ua` here, since returning a struct instead of a pointer will effectively do the copy.